recipe/meta.yaml

```diff
@@ -146,4 +146,6 @@ about:
 
 extra:
   recipe-maintainers:
-    - mjames-upc
+    - dopplershift
+    - dcamron
+    - srcarter3
```

Comment:
@dcamron, @srcarter3 are you okay with being maintainers?

Reply:
Yep!